supplementary/alica_ros1/alica_tests/src/test/test_alica_sync_transition.cpp

```diff
@@ -28,17 +28,10 @@ class AlicaSyncTransition : public AlicaTestMultiAgentFixture
 {
 protected:
     const int agentCount = 2;
-    const char* getRoleSetName() const override { return "RolesetTA"; }
+    const char* getRoleSetName(const int agentNumber) const override { return "RolesetTA"; }
```

Comment:
unused named argument, could be `const int /*agentNumber*/` if you want to keep a name (multiple places)

Reply:
I added this myself because if someone wants to have a roleset per agent you can (is this possible in this desirable? if it is not I can remove it), even though no test is using this right now.

Reply:
I mean that it's a potential warning message. If you should put argument but didn't need it it's better be nameless. But it's a good question why do we need it (and do we need it)?

Reply:
removed for now, until we actually need it